tuna/subcmd/parse_subtuna_args.py

```diff
@@ -0,0 +1,123 @@
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+###############################################################################
+"""Module to represent MIOpen subcommands parsers"""
+import jsonargparse
+from tuna.parse_args import TunaArgs, setup_arg_parser
+
+
+def get_import_cfg_parser(
+    with_yaml: bool = True) -> jsonargparse.ArgumentParser:
+  """Return parser for import_configs subcommand"""
+
+  parser = setup_arg_parser(
+      'Import MIOpenDriver commands and MIOpen performance DB entries.',
+      [TunaArgs.VERSION, TunaArgs.CONFIG_TYPE],
+      with_yaml=with_yaml)
+
+  group = parser.add_mutually_exclusive_group()
+
+  group.add_argument('--print_models',
+                     dest='print_models',
+                     action='store_true',
+                     help='Print models from table')
+  group.add_argument('--add_benchmark',
+                     dest='add_benchmark',
+                     action='store_true',
+                     help='Insert new benchmark')
+
+  parser.add_argument('-b',
+                      '--batches',
+                      type=str,
+                      dest='batches',
+                      help='Batch sizes to iterate over in the given configs')
+  parser.add_argument('--batchsize',
+                      dest='batchsize',
+                      type=int,
+                      default=None,
+                      required=False,
+                      help='Specify model batchsize')
+  parser.add_argument(
+      '-c',
+      '--command',
+      type=str,
+      dest='command',
+      default=None,
+      help='Command override: run a different command on the imported configs',
+      choices=[None, 'conv', 'convfp16', 'convbfp16'])
+  parser.add_argument('-d',
+                      '--driver',
+                      dest='driver',
+                      type=str,
+                      default=None,
+                      help='Specify driver cmd')
```

Comment:
This input seems to allow the user to input a single line command to be added as a config. In our import_cfgs.py implementation however it appears to only be implemented for add_benchmark. So we should leave this in the miopen file for now.